substrate/frame/balances/src/benchmarking.rs

```diff
@@ -297,6 +297,44 @@ mod benchmarks {
 		assert_eq!(Balances::<T, I>::total_issuance(), ti + delta);
 	}
 
+	// Benchmark `burn` extrinsic with the worst possible condition - burn kills the account.
```

Comment:
```suggestion
	/// Benchmark `burn` extrinsic with the worst possible condition - burn kills the account.
```
Should be possible i think.